prerequisites/software.rst

```diff
@@ -1,6 +1,60 @@
 Software
 ********
 
+1. Client requirements
+======================
+
+Please note that, while Zammad being a web application, there's some
+requirements for your clients. This ensures that Zammad works as expected.
+
+1.1. Supported Browsers
+-----------------------
+
+.. list-table:: Zammad/Browser version compatibility
+   :header-rows: 1
+   :widths: 20, 30
+
+   * - Firefox 78+
+     - (ESR)
+   * - (Google) Chrome 83+
+     - This also applies for all Chromium based Browsers like Microsoft Edge
+   * - Opera 69+
+     - (based on Chromium 83)
+   * - Microsoft Internet Explorer 11
+     - 
+   * - Safari 11
+     - 
+
+.. note::
+
+   | Zammad heavily uses Javascript which makes it a hard requirement.
+   |
+   | Addons that hook into page content may interfere with Zammads function
+     which is not a bug. 
+   | Google Chromes translation module is known to do
```

Comment:
Would it not be better to suppress browser-based translation of Zammad's UI? https://www.w3.org/International/questions/qa-translate-flag
Issue perhaps?

Reply:
I mean if you believe that this is a bug of Zammads UI?
Especially on Zammad versions where the language translation is fairly low, this may be an issue for agents.

(I mean we can't force people to translate Zammad, right?!)